### DISCUSSION TITLE
Suggestion : Colorcoding Of Affixes

### DISCUSSION THREAD

**Discussion by nezam7** [Lailloken/Exile-UI#303] [Ideas & Feedback]
![image](https://github.com/Lailloken/Lailloken-UI/assets/4648255/f4da2803-a923-47d4-9c2b-2c62cf294596)
Is it possible to do something like this? Not necessarily a user defined but at least a 2 color from suffixes/prefixes also would do wonder. 

taken idea from reddit post here. 
https://www.reddit.com/r/pathofexile/comments/1600v7q/qol_suggestion_user_defined_colorcoding_of_affixes/

**Comment by Lailloken** [comment]
The item-info feature lets you set specific affixes as (un)desired and highlights them accordingly. It's even more flexible than the reddit mockup because you can specify certain mods as (un)desired based on item-class, e.g. only highlight flat-ele damage on claws but not any other weapon-class.
![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/fdb2e9c6-5d48-4ba7-ae0a-4d51130f8d65)

The map-info feature lets you highlight map mods in up to four different colors, and also categorizes them and makes everything more compact.
![image](https://user-images.githubusercontent.com/61888437/260299445-d59f8c12-45df-4ac7-a1ab-1035020e9f89.png)
